### PR TITLE
Simplify EXTRATESTs

### PR DESCRIPTION
**Pull request by asdil12**
Ticket: https://progress.opensuse.org/issues/44252

Todo:
- [ ] Update toolkits testsuite: Prepend `prepare,` to `EXTRATEST` var

### CODE REVIEW

**Comment by asdil12** [comment]
@okurz WDYT?

File: products/opensuse/main.pm [hunk 1]
```diff
@@ -329,14 +329,6 @@ elsif (get_var("SYSCONTAINER_IMAGE_TEST")) {
...
     load_syscontainer_tests();
 }
-elsif (get_var('GNUHEALTH')) {
-    boot_hdd_image;
-    loadtest 'gnuhealth/gnuhealth_install';
-    loadtest 'gnuhealth/gnuhealth_setup';
-    loadtest 'gnuhealth/gnuhealth_client_install';
-    loadtest 'gnuhealth/gnuhealth_client_preconfigure';
-    loadtest 'gnuhealth/gnuhealth_client_first_time';
-}
```
[the remark this answers is not in the document]
Comment (asdil12) [reply products/opensuse/main.pm]: But then I won't be able to schedule it as an extratest.

[the remark this answers is not in the document]
Comment (asdil12) [reply products/opensuse/main.pm]: done anyway

**Comment by asdil12** [comment]
@okurz done

**Comment by michelmno** [comment]
could we have side effect of this PR as per issue#45902
https://progress.opensuse.org/issues/45902

**Comment by DimStar77** [comment]
Definitively a side effect for the extra tests to no longer pass on TW.

Guys, if you introduce new mandatory variables that were implied before: can we please, pretty please with sugar, make sure that those variables are being defined where they need to be set? You can't just run around and keep on introducing stuff like this without caring for fallouts (and the need of adding variables is apparently not unexpected - since it was mentioned in the initial comment, except that the todo list was utter garbage and incomplete)

**Pull request merged**